runtime/drivers/registry.go

```diff
@@ -92,6 +92,8 @@ type InstanceConfig struct {
 	// AlertStreamingRefDefaultRefreshCron sets a default cron expression for refreshing alerts with streaming refs.
 	// Namely, this is used to check alerts against external tables (e.g. in Druid) where new data may be added at any time (i.e. is considered "streaming").
 	AlertsDefaultStreamingRefreshCron string `mapstructure:"rill.alerts.default_streaming_refresh_cron"`
+	// MetricsComparisonsCTE indicates whether to rewrite metrics comparison queries to use CTEs.
+	MetricsComparisonsCTE bool `mapstructure:"rill.metrics.comparisons_cte"`
```

Comment:
I would be good with just making it part of the `rill.metrics.approximate_comparisons` option for now – not sure if people will need the ability to opt out of this